wordpress-mobile/WordPress-iOS
# Offline Mode: Create copes of deprecated classes

- Create copes of deprecated classes to make it easier to make changes.
- Remove state restoration support just in case. My understanding is that it never worked, and relying on Apple's state restoration is a bad idea ™  

To test:

- Enable `.syncPublishing` and verify that new classes are used
- Disable `.syncPublishing` and verify that old classes are used

## Regression Notes
1. Potential unintended areas of impact: Post & Page lists
2. What I did to test those areas of impact (or what existing automated tests I relied on): n/a
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


Testing checklist:
